jlh/JonFirstApp/app/src/main/java/com/e_conomic/jonfirstapp/MainActivity.java

```diff
@@ -139,7 +139,7 @@ private void changeFragmentLayout(View fragment_layout, int newLayoutParam) {
         fragment_layout.setLayoutParams(layoutParams);
     }
 
-    /** Retrieves the isDisplayMessageFragmentVisible boolean. */
+    /** Check whether the fragment that displays the message is visible. */
```

Comment:
Better. Common jargon could be something like "Returns true if this fragment is visible"